src/main/java/at/hannibal2/skyhanni/config/features/inventory/SackDisplayConfig.java

```diff
@@ -86,12 +86,12 @@ public String toString() {
     public SortingTypeEntry sortingType = SortingTypeEntry.DESC_STORED;
 
     public enum SortingTypeEntry implements HasLegacyId {
```

Comment:
Now might be a good time to broaden the scope of this enum class, and merge it with the one in [ChestValueConfig](https://github.com/hannibal002/SkyHanni/blob/6e176ae0dae3b991f77287507f734de8cbe5b860/src/main/java/at/hannibal2/skyhanni/config/features/inventory/ChestValueConfig.java#L61).

Reply:
good idea

Reply:
this cant be done easily. idk if the chest value display even needs amount sorting, but the sacks display needs it.
![image](https://github.com/user-attachments/assets/d6e8bdd1-587b-449d-9b33-2ee450c93be1)